api/src/routes/edit.ts

```diff
@@ -83,5 +84,13 @@ async function getChildren(req, res) {
 
 edit.get("/object/children", requireToken, getChildren);
 edit.get("/object/children/:pid", requireToken, pidSanitizer, getChildren);
+edit.get("/object/details/:pid", requireToken, pidSanitizer, async function (req, res) {
+    const pid = req.params.pid;
+    const obj = FedoraObject.build(pid);
+    const sort = await obj.getSort();
+    const metadata = await Fedora.getInstance().getDC(pid);
```

Comment:
What does DC mean? Is it possible to have a more descriptive function?

Reply:
DC stands for [Dublin Core](https://dublincore.org/), the metadata standard used by the digital library. I wouldn't object to renaming this method to getDublinCore for clarity if you wish, though that's probably best handled in a separate PR.

Reply:
Okay.